share/wake/lib/system/environment.wake

```diff
@@ -13,9 +13,12 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+
 package wake
 
-def uname = prim "uname"
+def uname =
+    prim "uname"
```

Comment:
We have some polarizing choices for `=` and `\n`, being:
* Zero chars after `=`, then newline
* Up to line-length after `=`, then newline

I can try to come up with some softer in-betweens:

We could have a: 
* Max of (say) 20 chars after `=` (up to regular line length), then newline

or a even a 
* Only allow `[a-zA-Z0-9_.]` after `=`, then newline (ie, only 'simple' looking expressions expression allowed)